facade-app/src/test/CommandInjection.test.tsx

```diff
@@ -4,11 +4,13 @@ import { render, screen, waitFor } from "@testing-library/react";
 import { Content } from "../Components/Content";
 import testFixture from "./fixtures";
 
-const actual = jest.requireActual("../Utilities/Utils");
-const utilMocks = {
-  ...actual,
-  getResource: jest.fn()
-}
+jest.mock('../Utilities/Utils', () => {
```

Comment:
When i run `npm run test`, i still see the BE call failures.
```
 console.error
      Error: Error: connect ECONNREFUSED 127.0.0.1:80
          at Object.dispatchError (/Users/pocketpills/MyLearnings/opensource/VulnerableApp-facade/facade-app/node_modules/jsdom/lib/jsdom/living/xhr/xhr-utils.js:63:19)
          at Request.<anonymous> (/Users/pocketpills/MyLearnings/opensource/VulnerableApp-facade/facade-app/node_modules/jsdom/lib/jsdom/living/xhr/XMLHttpRequest-impl.js:655:18)
          at Request.emit (node:events:538:35)
          at ClientRequest.<anonymous> (/Users/pocketpills/MyLearnings/opensource/VulnerableApp-facade/facade-app/node_modules/jsdom/lib/jsdom/living/helpers/http-request.js:121:14)
          at ClientRequest.emit (node:events:526:28)
          at Socket.socketErrorListener (node:_http_client:442:9)
          at Socket.emit (node:events:526:28)
          at emitErrorNT (node:internal/streams/destroy:157:8)
          at emitErrorCloseNT (node:internal/streams/destroy:122:3)
          at processTicksAndRejections (node:internal/process/task_queues:83:21) undefined
```
Also there are fewer warnings, is it possible to fix them?